src/pages/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react"
+import { Link, useNavigate } from "react-router-dom";
+import FormInput from "../../components/FormInput/FormInput";
+import { signIn } from "../../redux/slices/userSlice";
+import { useAppDispatch, useAppSelector } from "../../hooks/hooks";
+import './SignIn.css';
+
+const SignIn = () => {
+  const [values, setValues] = useState({
+    email: "",
+    password: ""
+  });
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+
+  const usersFromStore = useAppSelector(state => state.user);
```

Comment:
```suggestion
  const users = useAppSelector(state => state.user);
```
избыточный нейминг. И не понятно, если users - объект с пользователями, то почему state.user - единственное число.